ReactSerial/SerialHOC.js

```diff
@@ -148,11 +153,9 @@ const withSerialCommunication = (WrappedComponent) => {
       return (
         <React.Fragment>
           <WrappedComponent
-            ipcAvailable={ipcAvailable}
             sendData={this.sendData}
             setOnDataCallback={this.setOnDataCallback}
-            startIpcCommunication={this.startIpcCommunication}
-            stopIpcCommunication={this.stopIpcCommunication}
```

Comment:
The `startIpcCommunication` and `stopIpcCommuncication` methods are needed further down the line to ensure apps can force `stele` to restart ipc communication.